include/gc/Transforms/Microkernel/BrgemmRuntimeUtils.h

```diff
@@ -0,0 +1,51 @@
+//===-- BrgemmRuntimeUtils.h - Utils for Brgemm Runtime ---------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef GC_MICROKERNEL_BRGEMMRUNTIMEUTILS_H
+#define GC_MICROKERNEL_BRGEMMRUNTIMEUTILS_H
+
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/PatternMatch.h"
+#include "oneapi/dnnl/dnnl_types.h"
```

Comment:
Is this a base toolkit dependency?

Reply:
No, it just contains data types enum used by dnnl, and that's all what we need.